app/views/posts/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1>This page shows all posts</h1>
\ No newline at end of file
+<main>
+    <div class="posts-main-container">
+        <div class="single-user">
+            <div class="user-image">
+                <img src="<%= @user.photo %>" alt="profile" class="user-profile">
+            </div>
+            <div class="user-information">
+                <h2 class="user-name"><%= @user.name %></h2>
+                <span class="post-counter"><p class="count">Number of posts: <%= @user.posts.count %></p></span>
+            </div>
+        </div>
+        <div class="user-all-posts">
+            <% @posts.each_with_index do |post, index| %>
+                <div class="user-post">
+                    <h4 class="post">Post #<%= index+1 %></h4>
+                    <div class="post-content">
+                        <p class="post-paragraph">
+                            <%= post.text %>
+                        </p>
+                        <span class="posts-count"><p class="count">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p></span>
+                    </div>
+                </div>
+                <div class="posts-comment">
+                    <% post.comments.each do |comment| %>
+                        <p class="comments"><%= comment.author.name %> : <%= comment.text %></p>
+                    <% end %>
```

Comment:
- [ ] As per the requirement, only the 5 most recent comments for the post must be rendered here. Kindly implement this.
![image](https://github.com/Javed-Shahbaz/Blog-App/assets/91486791/97d4dc5b-c218-40db-88a1-838421286ee7)
(your model)